packages/components/src/button/Button.spec.js

```diff
@@ -63,4 +63,11 @@ describe('Button', () => {
     wrapper.simulate('click');
     expect(props.onClick).toBeCalled();
   });
+
+  it('tests work of custom CSS classes passed via className prop', () => {
```

Comment:
How about simplifying it a bit?
```suggestion
  it('adds passed class name to existing class names', () => {
```